app/views/datasets/_contact.html.erb

```diff
@@ -1,9 +1,11 @@
 <%
-  if contact_email_is_email?(dataset)
-    mail = mail_to(contact_email_for(dataset), nil, {'data-ga-event': 'contact', 'data-ga-publisher': @dataset.organisation.name, class: 'govuk_link'})
-  else
-    mail = link_to(contact_email_for(dataset), contact_email_for(dataset), {'data-ga-event': 'contact', 'data-ga-publisher': @dataset.organisation.name, class: 'govuk_link'})
-  end
+  default_name = "Contact #{@dataset.organisation.title} regarding this dataset"
```

Comment:
Could you change the variable name to something like default_message to better represent what it contains? 

I'm also wondering about whether it should be bare text in the code - generally we try to put strings into the translation files, and I know it's possible to pass a parameter in too, though I can't remember the syntax. Don't worry about it too much if there are other examples of bare strings, though. Likewise for examples further down this PR.